stackslib/src/net/api/getsortition.rs

```diff
@@ -111,6 +113,9 @@ pub struct SortitionInfo {
     /// In Stacks 2.x, this is the winning block.
     /// In Stacks 3.x, this is the first block of the parent tenure.
     pub committed_block_hash: Option<BlockHeaderHash>,
+    #[serde(with = "prefix_opt_hex")]
+    /// doc
```

Comment:
I'll need help to write this documentation

Reply:
```rust
/// This is the VRF seed generated by this sortition
```